core/tests/unit/ldml/test_transforms.cpp

```diff
@@ -1030,6 +1051,62 @@ test_normalize() {
     assert_marker_map_equal(map, expm);
   }
 
+  // doubled markers tests
+  {
+    marker_map map;
+    std::cout << __FILE__ << ":" << __LINE__ << " - doubled marker1 " << std::endl;
+    const std::u32string src        = U"e\uffff\u0008\u0001\u0300\u0320\u0300";
+    const std::u32string expect_rem = U"e\u0300\u0320\u0300";
+    const std::u32string expect_nfd = U"e\u0320\uffff\u0008\u0001\u0300\u0300";
+    auto dst_rem                    = remove_markers(src, &map);
```

Comment:
Refer to https://github.com/keymanapp/keyman/pull/10541#discussion_r1472194048 - do we also have a test with a marker in front of the _second_ rather than the first?

Reply:
added these.